spacy/pipeline/pipes.pyx

```diff
@@ -202,7 +202,7 @@ class Pipe(object):
         serialize["cfg"] = lambda p: srsly.write_json(p, self.cfg)
         serialize["vocab"] = lambda p: self.vocab.to_disk(p)
         if self.model not in (None, True, False):
-            serialize["model"] = lambda p: p.open("wb").write(self.model.to_bytes())
+            serialize["model"] = self.model.to_disk
```

Comment:
While it's technically the same, personally I would prefer to stick to the `lambda` style of notation just for consistency (i.e. `lambda p: self.model.to_disk(p)`)

Reply:
ok, changed it :)

Reply:
Sure, I pushed the changes in a second commit.